community/libgphoto2/build

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh -e
+
+./configure \
+    --prefix=/usr \
+    --disable-nls \
+    --disable-rpath
+	
+make
+make DESTDIR="$1" install
+
+# Set path of libraries such that print-camera-list can find them
+# If LB_LIBRARY_PATH is already defined, just add a new path
+export LD_LIBRARY_PATH="$1/usr/lib${LD_LIBRARY_PATH:+:}$LD_LIBRARY_PATH"
```

Comment:
I think this can be changed to just `export LD_LIBRARY_PATH="$LD_LIBRARY_PATH:$1/usr/lib"`

Reply:
Yes, that's right. Thanks! 
Done!